README.md

```diff
@@ -1,12 +1,60 @@
 # lusid-python-tools
 Python tools for the LUSID
 
```

Comment:
There should be the LUSID banner at the top of this README. Can you try rebasing

Reply:
Done